tests/test_audiomanager.cpp

```diff
@@ -0,0 +1,51 @@
+/**
+ * @file
+ *
+ * @author feliwir
+ *
+ * @brief Set of tests to validate audio components
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include <audiomanager.h>
+#include <gtest/gtest.h>
+#include <win32localfilesystem.h>
+#ifdef BUILD_WITH_OPENAL
+#include <alaudiomanager.h>
+#endif
+
+extern LocalFileSystem *g_theLocalFileSystem;
```

Comment:
IIRC this was to prevent conflicts with the `g_theLocalFileSystem`  from the Thyme library. Since that library is staticaly linked into the tests this will define the symbol twice